t/Mock/Tilma.pm

```diff
@@ -10,6 +10,58 @@ has json => (
+              </gml:LinearRing>
+            </gml:exterior>
+          </gml:Polygon>
+        </ms:msGeometry>
+        <ms:fid>1000.000000000000000</ms:fid>
+        <ms:owner>Tilfen Land</ms:owner>
+        <ms:maintainby>Peabody</ms:maintainby>
+        <ms:managedby>Thamesmead Commercial Team</ms:managedby>
+      </ms:grass>
+    </gml:featureMember>
+</wfs:FeatureCollection>';
+
+my $thamesmead_asset_not_found = '<?xml version=\'1.0\' encoding="UTF-8" ?>
+<wfs:FeatureCollection
+   xmlns:ms="http://mapserver.gis.umn.edu/mapserver"
+   xmlns:gml="http://www.opengis.net/gml"
+   xmlns:wfs="http://www.opengis.net/wfs"
+   xmlns:ogc="http://www.opengis.net/ogc"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://mapserver.gis.umn.edu/mapserver https://tilma.staging.mysociety.org:80/mapserver/thamesmead?SERVICE=WFS&amp;VERSION=1.1.0&amp;REQUEST=DescribeFeatureType&amp;TYPENAME=hardsurfaces&amp;OUTPUTFORMAT=SFE_XMLSCHEMA  http://www.opengis.net/wfs http://schemas.opengis.net/wfs/1.1.0/wfs.xsd">
+   <gml:boundedBy>
+      <gml:Null>missing</gml:Null>
+   </gml:boundedBy>
+</wfs:FeatureCollection>';
```

Comment:
I'd be tempted to put these two variables inside the `/mapserver/thamesmead` route handler, since they're not used anywhere else and are specific to that route.

Reply:
As I was making other changes, I've done this.